types/aws-lambda/trigger/appsync-resolver.d.ts

```diff
@@ -64,6 +64,8 @@ export interface AppSyncResolverEvent<TArguments, TSource = Record<string, any>
     };
     prev: { result: { [key: string]: any } } | null;
     stash: { [key: string]: any };
+    typeName?: string;
+    fieldName?: string;
```

Comment:
What are these? `info` already seemingly has these and they're not covered by the tests.

Reply:
Sorry, I accidentally left these changes in the commit. These two fields are already available in `AmplifyGraphQlResolverEvent` and therefore, must not be added to `AppSyncResolverEvent`.

I reverted changes to this file.